### PR TITLE
Add github action matrix

### PR DESCRIPTION
**Pull request by bullder**

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+name: Tests
+
+on:
+  pull_request: null
```
Comment (jdecool) [inline .github/workflows/tests.yml:4]:
```suggestion
on: [push]
```

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+env:
+  # see https://github.com/composer/composer/issues/9368#issuecomment-718112361
+  COMPOSER_ROOT_VERSION: "dev-main"
```
Comment (jdecool) [inline .github/workflows/tests.yml:8]: Are you sure we need this ?

```suggestion
```

File: .github/workflows/tests.yml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+      - uses: actions/checkout@v2
+
+      # see https://github.com/shivammathur/setup-php
+      - uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php-version }}
+          coverage: none
+
+      # composer install cache - https://github.com/ramsey/composer-install
+      - uses: "ramsey/composer-install@v1"
```
Comment (jdecool) [inline .github/workflows/tests.yml:36]: This is useless because you will run `composer update` on l.41
So you can remove this line.

**Review by jdecool — CHANGES_REQUESTED** [review]
Hi @bullder, thanks for your contribution.
I've made few comments.

When I the test workflow, I've a lot of errors like:

> ==> An exception has been thrown in file /Users/jdecool/Workspace/StatsdBundle/src/Tests/Units/DependencyInjection/M6WebStatsdExtension.php on line 71:
> ==> Symfony\Component\DependencyInjection\Exception\RuntimeException: Unable to load YAML config files as the Symfony Yaml Component is not installed. in /Users/jdecool/Workspace/StatsdBundle/vendor/symfony/dependency-injection/Loader/YamlFileLoader.php:676

Can you fix that ?

**Comment by bullder** [comment]
> Can you fix that ?

done

File: .github/workflows/tests.yml [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+          - 4.4.0
+          - ^5.0
+        experimental: [false]
+        include:
+          - php-version: 7.1
+            symfony-version: 4.4.0
+            experimental: false
+          - php-version: 8.0
+            symfony-version: ^5.0
+            experimental: true
```
Comment (jdecool) [inline .github/workflows/tests.yml:25]: I suggest to remove this section as this package is currently not compatible with PHP 8.x branch.

```suggestion
```

File: composer.json [hunk 5]
```diff
@@ -30,7 +30,8 @@
         "m6web/coke": "~1.2",
         "m6web/symfony2-coding-standard": "~1.2",
         "symfony/console": "^4.4 || ^5.0",
-        "symfony/http-foundation": "^4.4 || ^5.0"
+        "symfony/http-foundation": "^4.4 || ^5.0",
+        "symfony/yaml": "^4.4 || ^5.0"
```
Comment (jdecool) [inline composer.json:34]: Can you move this two line in `require` section ?

You should also add `symfony/validator`

**Review by jdecool — COMMENTED** [review]
Just two other comments. And it should be OK for me 👍🏼

**Comment by jdecool** [comment]
Can you also remove the `.travis.yml` file as we'll now use Github actions and update the build badge to be linked to Github Actions ?

For a badge sample, you can see [this PR](https://github.com/BedrockStreaming/Tornado/pull/50).

File: README.md [hunk 6]
```diff
@@ -1,4 +1,4 @@
-StatsdBundle [![Build Status](https://travis-ci.org/M6Web/StatsdBundle.png?branch=master)](https://travis-ci.org/M6Web/StatsdBundle)
+StatsdBundle [![Build Status](https://github.com/BedrockStreaming/StatsdBundle/actions/workflows/tests.yml/badge.svg)](https://github.com/BedrockStreaming/StatsdBundle/actions/workflows/tests.yml)
```
Comment (Oliboy50) [inline README.md:1]: the image URL does not look like the one that was used [here](https://github.com/BedrockStreaming/Tornado/pull/50/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R4) 

but I don't know if it's a big deal 🤷

**Review by Oliboy50 — APPROVED** [review]

File: README.md (hunk 6 above)
Comment (jdecool) [reply README.md:1]: It seems to be the Github official way to add a badge now.

**Review by jdecool — APPROVED** [review]

**Comment by jdecool** [comment]
Thanks @bullder

**Pull request merged**